.gitlab-ci/molecule.yml

```diff
@@ -0,0 +1,19 @@
+---
+molecule_docker:
+  stage: unit-tests
+  # TODO: This should be replaced with a community built image
+  # Dockerfile for this image can be found here:
+  # https://gitlab.esss.lu.se/ics-docker/kubespray-molecule/blob/master/Dockerfile
+  image: registry.esss.lu.se/ics-docker/kubespray-molecule:latest
+  tags:
+    - packet
+  variables:
+    MOLECULE_KUBEVIRT_SERVICE_TYPE: 'ClusterIP'
+  before_script:
+    - tests/scripts/rebase.sh
+  script:
+    - cd roles/container-engine/docker
+    - molecule test --all
+  # only:
+  #   changes:
+  #     - roles/container-engine/docker/**/*
```

Comment:
I think this should also run if .gitlab-ci/molecule.yml is changed

Reply:
I can add it (and also molecule/kubevirt/*) if you want to make sure to test role/s) upon changes to the CI configuration.

Reply:
I added files related to the CI pipeline and vm cration to the list.

Reply:
AFAIK `only: changes: ...`  does not work in our setup since we do not use GitLab merge requests but GitHub pull requests. The GitLab doc has a paragraph explaining why: https://docs.gitlab.com/ee/ci/yaml/#using-onlychanges-with-pipelines-for-merge-requests

Therefore I suggest to remove that (it's already commented) and run it all the time.